### PR TITLE
Use Client SDK for Rsocket gateway tests

### PR DESCRIPTION
**Pull request by snripa**

### CODE REVIEW

File: Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java [hunk 1]
```diff
@@ -1,130 +1,86 @@
...
   public void shouldReturnExceptionWhenQualifierIsWrong() {
-    final Mono<ServiceMessage> result = client.requestResponse(WRONG_QUALIFIER);
-
+    extention.shutdownServices();
+    Mono<String> result = service.one("hello");
     StepVerifier.create(result)
         .expectErrorMatches(throwable -> throwable.getMessage().startsWith("No reachable member with such service"))
-        .verify(VERIFY_DURATION);
+        .verify(TIMEOUT);
+    extention.startServices();
```
Comment (segabriel) [inline Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java]: what is it at the end? What does the test verify?

File: Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java [hunk 2]
```diff
@@ -1,130 +1,86 @@
...
   public void shouldReturnManyResponses() {
-    final Flux<ServiceMessage> result = client.requestStream(GREETING_MANY_STREAM);
+    Flux<Long> result = service.manyStream(5L);
 
     StepVerifier.create(result)
-      .assertNext(serviceMessage -> assertEquals("0", decodeData(serviceMessage.data())))
-      .expectNextCount(4)
-      .expectComplete()
-      .verify(VERIFY_DURATION);
+        .assertNext(next -> assertTrue(0 == next))
```
Comment (segabriel) [inline Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java]: use `assertEquals()` or some loop with generating wished responses

File: Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java [hunk 3]
```diff
@@ -1,130 +1,86 @@
...
-        .verify(VERIFY_DURATION);
-  }
-
-  private String decodeData(ByteBuf data) {
-    try {
-      return data.toString(StandardCharsets.UTF_8);
-    } finally {
-      ReferenceCountUtil.safeRelease(data);
-    }
+        .expectErrorMatches(t -> "Expected service request data of type: class java.lang.String, but received: null"
```
Comment (segabriel) [inline Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java]: I think our GW should response with the special format, what is it?

File: Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java (hunk 1 above)
Comment (snripa) [reply Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java]: the error content

Comment (snripa) [reply Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java]: need to stop/start service to emulate case with wrong qualifier of request

File: Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java (hunk 3 above)
Comment (snripa) [reply Tests/src/test/java/io/scalecube/gateway/rsocket/RSocketWebsocketServerTest.java]: this is what it replies(

File: Benchmarks/src/main/resources/scripts/cheetsheet [hunk 4]
```diff
@@ -9,7 +9,7 @@ sudo java -agentpath:/usr/local/YourKit-JavaProfiler-2018.04/bin/linux-x86-64/li
 ### Tests ###
 
 ## Request One ##
-sudo java -agentpath:/usr/local/YourKit-JavaProfiler-2018.04/bin/linux-x86-64/libyjpagent.so -cp scenarios.jar  io.scalecube.gateway.benchmarks.rsocket.RSWSRequestOneBenchmark gw_address=$GATEWAY_IP:8080 injectors=1250 messageRate=12500 executionTaskDurationInSec=300 rampUpDurationInSec=150
+sudo java -agentpath:/usr/local/YourKit-JavaProfiler-2018.04/bin/linux-x86-64/libyjpagent.so -cp scenarios.jar  io.scalecube.gateway.benchmarks.rsocket.remote.RemoteRequestOneBenchmark gw_address=$GATEWAY_IP:8080 injectors=1250 messageRate=12500 executionTaskDurationInSec=300 rampUpDurationInSec=150
```
Comment (segabriel) [inline Benchmarks/src/main/resources/scripts/cheetsheet]: so maybe need to change `gw_address` on `gatewayAddress`

**Pull request merged**